.github/workflows/release.yml

```diff
@@ -5,7 +5,7 @@ name: Release
 on:
   push:
     tags:
-      - 'v*'
```

Comment:
haven't used the tags keyword, but do we want to keep the 'v' character here? Looks like it was there to ensure that this workflow is only run for tags that start with v (versioning tags in other words, like v1.0.1 etc).

Reply:
Good point, removed that bit to add more flexibility but looks like all the tags that are there start with v.